charts/keess/values.yaml

```diff
@@ -0,0 +1,104 @@
+# Default values for keess.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: image-registry.powerapp.cloud/keess/keess
```

Comment:
What publishes this image?

Reply:
@benlangfeld I just changed the chart to point to the image generated by Jenkins.